static/js/components/ConfirmDeletion.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import Button from 'react-mdl/lib/Button';
+
+export default class ConfirmDeletion extends React.Component {
+  static propTypes = {
+    close:        React.PropTypes.func,
+    deleteEntry:  React.PropTypes.func,
+    open:         React.PropTypes.bool,
+  };
+
+  render () {
+    const { close, deleteEntry, open } = this.props;
+    let actions = [
+      <Button
+        type='button'
+        key='close'
+        className="cancel-button"
+        onClick={close}>
+        No
+      </Button>,
+      <Button
+        key='delete'
+        type='button'
+        className="save-button"
+        onClick={deleteEntry}>
```

Comment:
Can you change `deleteEntry` to not close the dialog itself? This handler could then be changed to call `deleteEntry` then `close` afterwards